pkg/network/multus.go

```diff
@@ -79,6 +79,95 @@ func cleanUpMultusOldName(ctx context.Context, client k8sclient.Client) []error
 	return []error{}
 }
 
+// cleanUpKubevirtIpamController checks specific kic outdated objects or ones that are no longer compatible and deletes them.
+func cleanUpKubevirtIpamController(conf *cnao.NetworkAddonsConfigSpec, ctx context.Context, client k8sclient.Client) []error {
```

Comment:
could you move this code to `pkg/network/kubevirt_ipam_controller.go`?

Reply:
oops done

lets see it pass